pkg/tags/tags.go

```diff
@@ -26,17 +26,14 @@ import (
 	"sync"
 	"time"
 
-	"github.com/ethersphere/bee/pkg/sctx"
 	"github.com/ethersphere/bee/pkg/swarm"
 )
 
 var (
 	TagUidFunc  = rand.Uint32
-	ErrNotFound = errors.New("tag not found")
+	NotFoundErr = errors.New("tag not found")
```

Comment:
keep as `ErrNotFound`, it is the same convention as in other packages

Reply:
done